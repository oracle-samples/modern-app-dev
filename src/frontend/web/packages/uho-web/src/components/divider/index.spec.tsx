/*
 * Copyright (c) 2023 Oracle and/or its affiliates.
 * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracl.com/licenses/upl/
 */
import { Divider } from '.';
import { h } from 'preact';
import { snapshot } from '../../../test/jetHelper';

snapshot('Divider', <Divider />);
