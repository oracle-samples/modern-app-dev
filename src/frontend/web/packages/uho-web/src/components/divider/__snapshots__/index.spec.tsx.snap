// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests for Divider Snapshot: Component - Divider 1`] = `
<DocumentFragment>
  <div
    data-oj-binding-provider="preact"
  >
    <div
      class="oj-sm-padding-4x-top divider"
    />
  </div>
  <div
    id="__oj_layerhost_container"
  />
</DocumentFragment>
`;
